addons/website_event_booth/controllers/event_booth.py

```diff
@@ -139,7 +139,7 @@ def _prepare_booth_registration_partner_values(self, event, kwargs):
         if request.env.user._is_public():
             contact_email_normalized = tools.email_normalize(kwargs['contact_email'])
             if contact_email_normalized:
-                partner = event._partner_find_from_emails_single(
+                partner = event.sudo()._partner_find_from_emails_single(
```

Comment:
LGTM :+1:
The commit message is really clear to me and well contextualized. I think we should indeed put the `sudo` call here on the caller. With the sudo'ed record, we just read the partner `id`, `name`, `email` and `phone` as fallback values. Apart from that, we don't do anything crazy with the record.